addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -0,0 +1,430 @@
+
+					case "310MHz":
+					case "315MHz":
+					case "433.92MHz receiver only":
+					default:
+						throw new IllegalArgumentException(
+								"Illegal tranceiver type");
+					}
+				}
+				break;
+
+			case RFXComBindingConstants.BRIDGE_TYPE_MANUAL_BRIDGE:
+				if (conf.transceiverType != null) {
+					switch (conf.transceiverType) {
+					case "433.92MHz":
+						msg.transceiverType = TransceiverType._443_92MHZ_TRANSCEIVER;
+						break;
+					case "433.92MHz receiver only":
+						msg.transceiverType = TransceiverType._443_92MHZ_RECEIVER_ONLY;
+						break;
+					case "310MHz":
+						msg.transceiverType = TransceiverType._310MHZ;
+						break;
+					case "315MHz":
+						msg.transceiverType = TransceiverType._315MHZ;
+						break;
+					default:
+						throw new IllegalArgumentException(
+								"Illegal tranceiver type");
+					}
+				}
+				break;
+
+			default:
+				throw new IllegalArgumentException("Illegal tranceiver type");
+			}
+
+			msg.enableUndecodedPackets = configuration.enableUndecoded;
+			msg.enableImagintronixOpusPackets = configuration.enableImagintronixOpus;
+			msg.enableByronSXPackets = configuration.enableByronSX;
+			msg.enableRSLPackets = configuration.enableRSL;
+			msg.enableLighting4Packets = configuration.enableLighting4;
+			msg.enableFineOffsetPackets = configuration.enableFineOffsetViking;
+			msg.enableRubicsonPackets = configuration.enableRubicson;
+			msg.enableAEPackets = configuration.enableAEBlyss;
+			msg.enableBlindsT1T2T3T4Packets = configuration.enableBlindsT1T2T3T4;
+			msg.enableBlindsT0Packets = configuration.enableBlindsT0;
+			msg.enableProGuardPackets = configuration.enableProGuard;
+			msg.enableLaCrossePackets = configuration.enableLaCrosse;
+			msg.enableHidekiUPMPackets = configuration.enableHidekiUPM;
+			msg.enableADPackets = configuration.enableADLightwaveRF;
+			msg.enableMertikPackets = configuration.enableMertik;
+			msg.enableVisonicPackets = configuration.enableVisonic;
+			msg.enableATIPackets = configuration.enableATI;
+			msg.enableOregonPackets = configuration.enableOregonScientific;
+			msg.enableMeiantechPackets = configuration.enableMeiantech;
+			msg.enableHomeEasyPackets = configuration.enableHomeEasyEU;
+			msg.enableACPackets = configuration.enableAC;
+			msg.enableARCPackets = configuration.enableARC;
+			msg.enableX10Packets = configuration.enableX10;
+
+			return msg.decodeMessage();
+		}
+
+		throw new IllegalArgumentException("");
+	}
+
+	public synchronized void sendMessage(RFXComMessage msg) {
+
+		((RFXComBaseMessage) msg).seqNbr = getNextSeqNumber();
+		byte[] data = msg.decodeMessage();
+
+		logger.debug("Transmitting message '{}'", msg);
+		logger.trace("Transmitting data: {}",
+				DatatypeConverter.printHexBinary(data));
+
+		setResponseMessage(null);
+
+		try {
+			connector.sendMessage(data);
+		} catch (IOException e) {
+			logger.error("Message sending to RFXCOM controller failed.", e);
```

Comment:
instead of logging, better set the status to OFFLINE and provide the exception message as a status details